go.mod

```diff
@@ -1,41 +1,20 @@
 module github.com/markbates/goth
```

Comment:
thanks for updating this PR. It looks like somethings have been reverted from master in go.mod. Could you bump the version to 1.18, and run `go mod tidy`?

Reply:
should be ok now.